requirements-app.txt

```diff
@@ -14,8 +14,8 @@ eventlet==0.25.1
 gunicorn==19.7.1  # pyup: ignore, >19.8 breaks eventlet patching
 iso8601==0.1.12
 jsonschema==3.0.2
-marshmallow-sqlalchemy==0.17.0
-marshmallow==2.20.2  # pyup: <3 throws errors
+marshmallow-sqlalchemy==0.17.2
+marshmallow==3.0.1  # pyup: <3 throws errors
```

Comment:
wonder why this doesn't work like the `celery==3.1.26.post2 # pyup: <4` line above